scripts/helpers/engine.js

```diff
@@ -2,6 +2,15 @@
 
 'use strict';
 
+const crypto = require('crypto');
+
+hexo.extend.helper.register('gitalk_md5', function(path) {
+  var str = this.url_for(path);
+  str = encodeURI(str);
```

Comment:
This line should be removed due to https://github.com/hexojs/hexo/pull/3708

`post.path` is encoded twice after Hexo 4.0.0